packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNodeUtils.ts

```diff
@@ -70,100 +69,33 @@ export class EscapedPath {
 	public static create(path: string): EscapedPath {
 		return new EscapedPath(encodeURIComponent(path));
 	}
-	public static createAndConcat(pathParts: string[]): EscapedPath {
-		let ret = EscapedPath.create(pathParts[0] ?? "");
-		for (let i = 1; i < pathParts.length; i++) {
-			// Non null asserting here since we are iterating over pathParts
-			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-			ret = ret.concat(EscapedPath.create(pathParts[i]!));
-		}
-		return ret;
-	}
 	public toString(): string {
 		return this.path;
 	}
+	public concatWith(path: EscapedPath, concatWith: string): EscapedPath {
+		return new EscapedPath(`${this.path}/${encodeURIComponent(concatWith)}/${path.path}`);
+	}
 	public concat(path: EscapedPath): EscapedPath {
```

Comment:
How's this different from concatWith above? can we highlight the difference in the signature?

Reply:
removed unused function concat